servo-core/src/main/java/com/netflix/servo/monitor/StatsMonitor.java

```diff
@@ -287,6 +310,20 @@ public String toString() {
 
   /**
    * Creates a new instance of the timer with a unit of milliseconds,
+   * using the {@link ScheduledExecutorService} provided by the user,
+   * and the default Clock.
```

Comment:
Can you add a note about the need to call `setRemoveOnCancelPolicy` on the executor?